lib/index.js

```diff
@@ -276,13 +300,25 @@ Appboy.prototype.page = function(page) {
   if (!settings.trackAllPages && !settings.trackNamedPages) return;
   if (settings.trackNamedPages && !page.name()) return;
 
-  var userId = page.userId();
+  var userId = this.analytics.user().id();
```

Comment:
@ladanazita @ucarion @viparthasarathy do cases exist where `this.analytics.user()` is undefined?  I remember having to guard against that in some cases

Reply:
There should be no need to guard against `user()` being undefined in this case. In `initialize`, a guard would be required, however.